front/lib/api/assistant/actions/utils.ts

```diff
@@ -56,18 +56,11 @@ export const ACTION_SPECIFICATIONS: Record<
     dropDownIcon: TableIcon,
     flag: null,
   },
-  WEBSEARCH: {
-    label: "Web search",
-    description: "Perform a web search",
+  WEBSEARCH_AND_BROWSE: {
+    label: "Web search and browse",
+    description: "Perform a web search and browse the results",
```

Comment:
I think there is a product question here; just browsing a given url is a legit use case, so we should make it clear to users -- not in the direct scope of this PR but ccing @Duncid and @spolu who were part of this discussion and @gabhubert as copy master

E.g. I am on a website, don't want to read it, copy paste the url to the assistant and tell him to read it for me
=> user has to understand that this action covers the use case, so we need the right copy here